FordLabs/PeopleMover
# Reset viewing date on space dashboard render

Co-authored-by: Vianney Willot <vianney.willot@gmail.com>
Co-authored-by: Elise Griffiths <egriff38@ford.com>

## Issue
Resolves #511

## What was done
- [x] Global "Viewing Date" gets reset now whenever you navigate to the space dashboard, which should address the bug where you could switch spaces and still see the same (non-today) date.

## How to test
1. Go to a space and modify the date you are viewing
2. Go to another space, and see that the date switched back to whatever today happens to be. 
3. Go back to the first space and see that it also does not rememeber what date you were viewing

#### *No accessibility features were added or addressed in this story*

### FOR DEVS
Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/

Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/

### FOR APPROVAL
Approval URL: https://approvalbranchui.apps.pd01i.edc1.cf.ford.com/

Approval Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20ApprovalBranch/
